### PR TITLE
UI: Setup Ingredient Storage

### PR DESCRIPTION
**Pull request by AkshatPandey1**
image
image

### CODE REVIEW

**Comment by steventango** [comment]
@AkshatPandey1 can you switch over to using the common DeleteButton, this way we have less documentation to do. Also, note the other common elements used in MealPlan code and consider using them to increase code reuse.

File: app/src/main/java/com/example/wellfed/ActivityBase.java [hunk 1]
```diff
@@ -16,4 +16,8 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
         return super.onOptionsItemSelected(item);
     }
+
+    public abstract void deleteIngredient();
+
+    public abstract void delete();
```
Comment (steventango) [inline app/src/main/java/com/example/wellfed/ActivityBase.java:22]: revert this, and all the uses in the subclasses

**Review by steventango — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/wellfed/ActivityBase.java (hunk 1 above)
Comment (AkshatPandey1) [reply app/src/main/java/com/example/wellfed/ActivityBase.java]: I've made those changes, can you check it now?

Comment (steventango) [reply app/src/main/java/com/example/wellfed/ActivityBase.java]: LGTM

**Review by steventango — APPROVED** [review]

**Pull request merged**